src/Monolog/ErrorHandler.php

```diff
@@ -58,9 +58,6 @@ public function __construct(LoggerInterface $logger)
      */
     public static function register(LoggerInterface $logger, $errorLevelMap = [], $exceptionLevelMap = [], $fatalLevel = null): self
     {
-        //Forces the autoloader to run for LogLevel. Fixes an autoload issue at compile-time on PHP5.3. See https://github.com/Seldaek/monolog/pull/929
-        class_exists('\\Psr\\Log\\LogLevel', true);
-
```

Comment:
I removed also this... I guess the bug from back then it is fixed now.. isn't it?

Reply:
Yup good catch.